configs/mainnet/altair.yaml

```diff
@@ -45,10 +45,8 @@ ALTAIR_FORK_EPOCH: 18446744073709551615
 # ---------------------------------------------------------------
 # 1
 MIN_SYNC_COMMITTEE_PARTICIPANTS: 1
-# 2**13
-MAX_VALID_LIGHT_CLIENT_UPDATES: 8192
-# 2**13 (=8192)
-LIGHT_CLIENT_UPDATE_TIMEOUT: 8192
+# 2**64 - 1
+MAX_VALID_LIGHT_CLIENT_UPDATES: 18446744073709551615
```

Comment:
this value is just setting an upper bound on the possible size of a `ssz` list so i don't see a reason to not just use the max value (which matches the latest spec)

Reply:
nice catch

Reply:
Considering it's only used in a local store that max value is fine *but* considering this is just a local representation of a DB, there is no value in having it be an SSZ object (for merklization or consensus) and thus it does not need this list limit at all if we move to a more generic `dataclass`. This is what we did with the [fork choice `Store`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/fork-choice.md#store) and I think it appropriate here.